Liadi/events-manager
# #152796813 landing page template

#### What does this do?
 - Fix responsiveness issue
 - Fix proper spacing and sizing of elements issue
 - Fix Carousel (slide show) fixed dimension
 - Replace dull and unattractive images
 - Add advanced search fields (available on drop down)
 - Click Advanced search to hide/show fields

#### Task to be completed
 - Padding(top): Search button, for screen size <(400)
 - Styling
 
#### How to test features?
  - Navigate to https://liadi.github.io/events-manager/template/index.html
 - Test responsiveness using developer tools
 OR
 - Clone repo(for commit history)
 - Ensure branch is 'ch-landing-page-template-152796813' 
 - Open browser and 
 - Navigate to [repo base path]/template/index.html

#### Background context
 - None

#### Related PivotalTracker stories(id)
 - #152796813

#### Screenshots
-------------------------------------------------------------------------------
![land1](https://user-images.githubusercontent.com/29649308/32914196-0b1d8e2a-cb15-11e7-8018-6893a5fe460e.PNG)

-----------------------------------------------------------------------------
![land2](https://user-images.githubusercontent.com/29649308/32914197-0b8629a8-cb15-11e7-83b8-f641d260a6d0.PNG)

-----------------------------------------------------------------------------
![land3](https://user-images.githubusercontent.com/29649308/32914199-0be6e932-cb15-11e7-9be1-6875ac96057a.PNG)

-----------------------------------------------------------------------------
![land4](https://user-images.githubusercontent.com/29649308/32914200-0c56b140-cb15-11e7-9adf-e1f8a996acc5.PNG)

#### Questions
 - None
